bin/arist

```diff
@@ -1,8 +1,66 @@
 #!/usr/bin/env python3
-import sys
-from openquake.engine.impact import main_cmd
-if __name__ == '__main__':
+import argparse
+from openquake.engine.impact import main_cmd, trivial_callback
+
```

Comment:
Please don't use `argparse` directly. The engine is using `baselib.sap`. Copy from commands/engine.py